wwwroot/index.html

```diff
@@ -32,11 +32,90 @@
     <!-- Leaflet -->
     <script>L_PREFER_CANVAS = true;</script>
     <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
+    <script type="text/javascript" src="public/js/load.js"></script>
+    <script type="text/javascript">
+      /* load your stuff */
+      loadJS(["build/nationalmap.js"])
+        .then(function() {
+          ui = document.getElementById("ui");
+          setTimeout(function(){
+            ui.setAttribute("class", "loaded-ui"); 
+          }, 5000);   
```

Comment:
A fixed 5 second timeout here seems kinda suspect. What's it fore?

Reply:
It takes some time from finishing downloading js to ui loads. I got 5s from trying. Probably too long

Reply:
Sure, but that 5s will be wildly different on different systems. We should be able to tell when it's done loading rather than hard-coding a particular time. The webpack chunk thing may help here too.